ClosedXML/Excel/Ranges/XLRange.cs

```diff
@@ -760,9 +736,11 @@ private void TransposeMerged(Int32 squareSide)
                 RangeAddress.FirstAddress.RowNumber + squareSide - 1,
                 RangeAddress.FirstAddress.ColumnNumber + squareSide - 1);
 
-            foreach (IXLRange merge in Worksheet.Internals.MergedRanges.Where(Contains))
+            foreach (XLRange merge in Worksheet.Internals.MergedRanges.Where(Contains))
```

Comment:
Please use this style: `foreach (var merge in Worksheet.Internals.MergedRanges.Where(Contains).Cast<XLRange>())`